dev-lua/luassert/luassert-1.8.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Lua Assertions Extension"
+HOMEPAGE="http://olivinelabs.com/busted/"
+SRC_URI="https://github.com/Olivine-Labs/luassert/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86"
+IUSE="luajit test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-lua/say-1.3_p1[luajit=]
```

Comment:
if one of old dependencies does not have a use flag that you depend on, you have to use  `use dependency-default`

to this would be `>=dev-lua/say-1.3_p1[luajit(-)=]`
which means if the package does not have this flag, assume it's disable.

`(+)` would mean assume it's enabled, and is needed when some use-flags removed and package is configured same way as if it's enabled.


https://devmanual.gentoo.org/general-concepts/dependencies/